### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Jieiku**
I noticed your demo has been offline for a long time.

I fixed the theme files so that they can be hosted from a root domain or a subfolder, so for example:

`https://elegaanz.github.io/`
**or**
`https://elegaanz.github.io/course/`

previous because of the root path for the css and js file, this would never work from a subfolder, now it works either way.

I also pointed the demo to one I put up for now, you can merge it as is, or upload your own demo.

Hope this helps.

### CODE REVIEW

File: theme.toml [hunk 1]
```diff
@@ -3,9 +3,9 @@ description = "A zola theme designed for online courses or tutorials"
 license = "GPL-3.0"
 homepage = "https://github.com/elegaanz/zola-theme-course"
 # The minimum version of Zola required
-min_version = "0.15.0"
+min_version = "0.17.1"
```
Comment (elegaanz) [inline theme.toml:6]: Is there a change in your code that requires v0.17?

File: config.toml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # The URL the site will be built for
-base_url = "https://example.com"
+#base_url = "https://example.com"
+base_url = "https://zola-themes-demos.github.io/course"
```
Comment (elegaanz) [inline config.toml]: Is this change necessary?

File: theme.toml [hunk 3]
```diff
@@ -3,9 +3,9 @@ description = "A zola theme designed for online courses or tutorials"
...
 # An optional live demo URL
-demo = "https://c.gelez.xyz"
+demo = "https://zola-themes-demos.github.io/course/"
```
Comment (elegaanz) [inline theme.toml]: I would like to host the demo myself if possible. I migrated it to a new server recently, that's probably why it is broken, I will try to see how to fix it. Could you revert this change in the meantime please?

**Review by elegaanz — COMMENTED** [review]
Thank you! I left a few comments, but it looks good otherwise.

File: theme.toml (hunk 1 above)
Comment (Jieiku) [reply theme.toml:6]: The function used for SRI changed from get_file_hash to get_hash https://github.com/getzola/zola/pull/1962 So eventually this would get you when you upgrade, better to use the new function now.

File: config.toml (hunk 2 above)
Comment (Jieiku) [reply config.toml]: No, but if you set the base_url to the url of your demo site then it is easier to automate the deployment of your demo, I will revert this.

File: theme.toml (hunk 3 above)
Comment (Jieiku) [reply theme.toml]: sure no problem.

File: theme.toml (hunk 1 above)
Comment (elegaanz) [reply theme.toml:6]: Ok, thanks for the info!
